source/assets/stylesheets/typography.scss

```diff
@@ -1,3 +1,8 @@
-h1, h2, h3, h4, h5 {
+h1, h2, h3, h5 {
   font-weight: 400;
 }
+
+h4 {
```

Comment:
@pajasevi is it ok? I'm using deeply nested heading in my blogs and `h4` wasn't highlighted enough IMHO.